R/pkg/R/mllib.R

```diff
@@ -917,14 +922,14 @@ setMethod("spark.lda", signature(data = "SparkDataFrame"),
 # Returns a summary of the AFT survival regression model produced by spark.survreg,
 # similarly to R's summary().
 
-#' @param object A fitted AFT survival regression model
+#' @param object a fitted AFT survival regression model.
 #' @return \code{summary} returns a list containing the model's coefficients,
 #' intercept and log(scale)
 #' @rdname spark.survreg
 #' @export
 #' @note summary(AFTSurvivalRegressionModel) since 2.0.0
 setMethod("summary", signature(object = "AFTSurvivalRegressionModel"),
-          function(object, ...) {
+          function(object) {
```

Comment:
I really like the fact that we do not need to have `...` here in the func definition. But I'm not sure I understand why exactly - in generics.R we have `setGeneric("summary", function(object, ...) `, doesn't it mean we need to have  `...` here?